tools/src/test/scala/com/nvidia/spark/rapids/tool/qualification/QualificationSuite.scala

```diff
@@ -351,6 +352,42 @@ class QualificationSuite extends FunSuite with BeforeAndAfterEach with Logging {
     runQualificationTest(logFiles, "nested_type_expectation.csv")
   }
 
+  test("test different types in ReadSchema") {
+    val outpath = Array(
+      s"$logDir/eventlog_complex_dec_nested", // complex nested types with decimal
```

Comment:
I didn't want to add a bunch of event logs just to test parsing of complex types.  Why can't se just send in strings that simulate the values in the event logs?

Reply:
Updated the test. PTAL.